test/test_train_imagenet.py

```diff
@@ -164,10 +166,13 @@ def test_loop_fn(model, loader, device, context):
     return correct / total_samples
 
   accuracy = 0.0
+  writer = SummaryWriter(log_dir='/tmp/imagenet_tensorboard')
   for epoch in range(1, FLAGS.num_epochs + 1):
     model_parallel(train_loop_fn, train_loader)
     accuracies = model_parallel(test_loop_fn, test_loader)
-    accuracy = sum(accuracies) / len(accuracies)
+    print("Epoch: {}".format(epoch))
+    accuracy = mean(accuracies)
+    writer.add_scalar('Accuracy/test', accuracy, epoch)
     if FLAGS.metrics_debug:
```

Comment:
We should probably print the accuracy mean as well.

Reply:
Done